hieradata/common.yaml

```diff
@@ -351,6 +351,9 @@ govuk::apps::publishing_api::db::backend_ip_range: "%{hiera('environment_ip_pref
 govuk::apps::rummager::nagios_memory_warning: 4600
 govuk::apps::rummager::nagios_memory_critical: 4900
 
+govuk::apps::service_manual_publisher::http_username: "%{hiera('http_username')}"
+govuk::apps::service_manual_publisher::http_password: "%{hiera('http_password')}"
+
```

Comment:
Don't see any need for these here. That's only useful if you're using the value from other hiera files and modifying/extending it somewhere here, which you're not.